editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -482,7 +481,7 @@ const RowForBaseControl = React.memo((props: RowForBaseControlProps) => {
             showHiddenControl={props.showHiddenControl}
           />
         </div>
-        {when(isBaseIndentationLevel(props), dataPickerButtonData.DataPickerOpener)}
+        {/* {when(isBaseIndentationLevel(props), dataPickerButtonData.DataPickerOpener)} */}
```

Comment:
can this be removed? or is it kept for future ref? (the ones below too, line 620 and 924)